Archive/Imo/Imo1962Q4.lean

```diff
@@ -134,7 +134,7 @@ Again, the final theorem is now just gluing together our lemmas.
 -/
 theorem imo1962_q4' {x : ℝ} : ProblemEquation x ↔ x ∈ solutionSet :=
   calc
-    ProblemEquation x ↔ cos x ^ 2 + cos (2 * x) ^ 2 + cos (3 * x) ^ 2 = 1 := by rfl
+    ProblemEquation x ↔ cos x ^ 2 + cos (2 * x) ^ 2 + cos (3 * x) ^ 2 = 1 := rfl
```

Comment:
yeah, I by far didn't change all occurrences of `by rfl` to `rfl`, but didn't catch this one (presumably, because I only did `lake build` to check and that perhaps does not build the Archiv.)